app/javascript/spec/ansibleCatalog/ansiblePlaybookFields.test.jsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import configureStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
```

Comment:
this is surprising? Why do we need redux-thunk in specs?
Or rather, why not initialize the store the "normal way"?

Reply:
I will do some thorough refactoring later in other PR. I've got some other things i would like to change.